charts/cortex/templates/store-gateway/store-gateway-svc-headless.yaml

```diff
@@ -1,21 +0,0 @@
-{{- if eq .Values.config.storage.engine "blocks" -}}
-apiVersion: v1
-kind: Service
-metadata:
-  name: {{ template "cortex-store-gateway.headlessname" . }}
```

Comment:
Same comment as for the ingester. The statefulset needs a headless service - and this is actually being referenced from the non-zone-aware stateful set.

Reply:
We still need to restore this file - isn't that so? As it is used as headless for storegateway - at least the one not "zoned".